astropy/units/photometric.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+This module defines magnitude zero points and related photometric quantities.
+"""
+
+
+import numpy as _numpy
+from .core import UnitBase, def_unit
+
+from ..constants import si as _si
+from . import cgs, si, astrophys
+
+
+_ns = globals()
+
+def_unit(['Bol', 'L_bol'], _si.L_bol0, namespace=_ns, prefixes=False,
+         doc="Luminosity corresponding to absolute bolometric magnitude zero")
+def_unit(['bol', 'f_bol'], _si.L_bol0 / (4 * _numpy.pi * (10.*astrophys.pc)**2),
+         namespace=_ns, prefixes=False, doc="Irradiance corresponding to "
+         "appparent bolometric magnitude zero")
+def_unit(['ABflux'], 3631e-23 * cgs.erg * cgs.cm**-2 / si.s / si.Hz,
+         namespace=_ns, prefixes=False,
+         doc="AB magnitude zero flux density.")
+def_unit(['STflux'], 3631e-12 * cgs.erg * cgs.cm**-2 / si.s / si.AA,
```

Comment:
Here, I think you may be wrong. At least, I thought `STmag` was defined by `21.0` (it is in all the HST files!). So, even more reason to give a reference (from STScI of course).